extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/handler/SacnBridgeHandler.java

```diff
@@ -94,7 +80,7 @@ protected void updateConfiguration() {
         logger.debug("refresh mode set to always: {}", refreshAlways);
 
         super.updateConfiguration();
-        updateStatus(ThingStatus.ONLINE);
+        updateStatus(ThingStatus.OFFLINE);
```

Comment:
I think this change broke your test cases, can you have a look please? Also while you are at the tests have a look if the others are still working.